.github/workflows/check-backport.yml

```diff
@@ -0,0 +1,37 @@
+name: Stable Backport Check
+on:
+  issue_comment:
+  pull_request_target:
+    types:
+      - labeled
+
+jobs:
+  check-backport:
+    name: Check Backport
+    if: ( github.event.issue.pull_request != '' && contains(github.event.comment.body, '@jellyfin-bot check backport') && github.event.comment.author_association == 'MEMBER' ) || github.event.label.name == 'stable backport'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Notify as seen
+        uses: peter-evans/create-or-update-comment@v1.4.5
+        with:
+          token: ${{ secrets.JF_BOT_TOKEN }}
+          comment-id: ${{ github.event.comment.id }}
+          reactions: '+1'
+
+      - name: Checkout the latest code
+        uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.JF_BOT_TOKEN }}
+          fetch-depth: 0
+
+      - name: Perform test backport
+        run: |
+          git config --global user.name "Jellyfin Bot"
+          git config --global user.email "team@jellyfin.org"
+          CURRENT_BRANCH=$( git symbolic-ref --short HEAD )
+          git checkout -b nmaster origin/master
+          git merge --no-ff ${CURRENT_BRANCH}
```

Comment:
In case the target branch of the PR is master (i.e. most if not all of them) wouldn't you be trying to merging master with master?

Reply:
No the current_branch is the PR branch, not master.

Reply:
Turns out I was wrong and my commit messages were so shitty they hid it. I'mma try with `pull_request` rather than target.

Reply:
Turns out that none of the obvious solutions for using `git` to set `CURRENT_BRANCH` worked, but I found `${{ github.head_ref }}`/`${GITHUB_REF}` (with `origin` prefixed it seems) instead.